boards/shields/rtkmipilcdb00000be/Kconfig.defconfig

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_RTKMIPILCDB00000BE
+if DISPLAY
+
+config INPUT_INIT_PRIORITY
+	default 89
+
+# Enable MIPI DSI, as this display controller requires it.
+
+endif # DISPLAY
+
+if LVGL
+
+# Configure LVGL to use touchscreen with KSCAN API
+
+config INPUT
+	default y
+
+config INPUT_GT911_INTERRUPT
```

Comment:
Not needed. This Kconfig already defaults to disabled

Reply:
I fixed it